README.md

```diff
@@ -110,6 +110,44 @@ if Mix.env == :dev do
 end
 ```
 
+## Disabling ansi_enabled setting
```

Comment:
OK.  True will the default (and current implementation) and False will be the `:ignore` route.